whylog/tests/tests_front/tests_whylog_factory.py

```diff
@@ -0,0 +1,26 @@
+from mock import MagicMock, patch
+
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+from whylog.tests.utils import create_config
+
+
+def tests_whylog_factory():
+    yaml_config, config_path = create_config()
+
+    file_mock = MagicMock()
+    file_mock.return_value = yaml_config
+    with patch('yaml.load', file_mock, create=True):
+        log_reader, teacher_generator = whylog_factory(config_path)
+    teacher = next(teacher_generator)
+
+    # check if returned object classes
+    assert isinstance(log_reader, LogReader)
+    assert isinstance(teacher, Teacher)
+
+    # check other teachers
```

Comment:
Why do we test other teachers? Isn't it enough to check it once? Maybe remove 4 following lines?